.github/workflows/test.yml

```diff
@@ -12,8 +12,11 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+
   set-matrix:
-    runs-on: ubuntu-latest
+    # FIXME: set-matrix is lightweight, run it on any self-hosted machines for kernel-patches so we do
```

Comment:
It's not really clear to me what is there to fix here, can you make that a bit more clear?

Reply:
oh boy, I must have written this comment without thinking it through.

I think I marked this as a FIXME because once we refactor this workflow/job to be re-usable, we can differ setting on what runner to run on to the caller (and default to ubuntu-latest). Will make it more explicit here.